test/CMakeLists.txt

```diff
@@ -66,10 +69,12 @@ endif()
 add_custom_target(build-onedpl-tests
     COMMENT "Build all oneDPL tests")
 
+set(run-onedpl-tests-depends build-onedpl-tests)
+
 add_custom_target(run-onedpl-tests
     COMMAND "${CMAKE_CTEST_COMMAND}" --output-on-failure
     USES_TERMINAL
-    DEPENDS build-onedpl-tests
+    DEPENDS ${run-onedpl-tests-depends}
```

Comment:
I don't necessarily object to this change, but I don't understand the reasoning for it.  Is this an artifact of a reverted change which had something else here?

Reply:
This change had no sense. Reverted. Thank you for the comment. It probably was a result of few past changes and refactors combined together.